modules/installation-guide/partials/assembly_configuring-the-number-of-workspaces-a-user-can-run.adoc

```diff
@@ -0,0 +1,18 @@
+:parent-context-of-configuring-the-number-of-workspaces-a-user-can-run: {context}
+
+[id="configuring-the-number-of-workspaces-a-user-can-run_{context}"]
+= Configuring the number of workspaces a user can run
+
+:context: configuring-the-number-of-workspaces-a-user-can-run
+
+This article describes how to configure the number of workspaces a user can run simultaneously. 
+
+include::partial$proc_configuring-the-number-of-workspaces-a-user-can-run-using-the-operator.adoc[leveloffset=+1]
+
+ifeval::["{project-context}" == "che"]
+
+include::partial$proc_configuring-the-number-of-workspaces-a-user-can-run-using-configmap.adoc[leveloffset=+1]
```

Comment:
Don't we have a two includes in {project-context}" == "che"?

Reply:
Yes, that's expected. 2 includes for che, only the operator include for crw.